modules/integration/pom.xml

```diff
@@ -96,6 +96,11 @@
       <artifactId>zookeeper</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Perhaps a separate issue, but none of our tests have a an implementing slf4j module on the classpath. So, none of them have an actual logger to use. They are all using the default NOP logger, or they are pulling in one transitively from one of the dependencies (sometimes multiple).

We need to make sure we configure one for our test scope, and exclude anything being brought in transitively. (And file tickets upstream to make the slf4j impl dependencies have `<optional>true</optional>` so that we don't have to explicitly exclude them in the future.)